content/docs/intro/concepts/state.md

```diff
@@ -13,124 +13,81 @@ menu:
-and [Working with Amazon S3 buckets](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html). Also see [AWS Session](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/) for configuring credentials for this backend.
-
-{{< /choosable >}}
-
-{{< choosable cloud azure >}}
-<p></p>
-
-For **Azure Blob Storage**, see [Azure Setup]({{< relref "/docs/intro/cloud-providers/azure/setup" >}})
-and Microsoft's [Storage Blobs Quickstart](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli). This backend requires setting the environment variables `AZURE_STORAGE_ACCOUNT` and either `AZURE_STORAGE_KEY` or `AZURE_STORAGE_SAS_TOKEN`.
-
-{{< /choosable >}}
+- Robust state management, with transactional checkpointing for fault tolerance and recovery
+- Concurrent state locking to prevent corrupting your infrastructure state in a team environment
+- Full deployment history for auditing and rollback purposes
+- Encrypted state in transit and at rest
+- Managed encryption and key management for secrets
+- Secure access to cloud resource metadata, with client-side authentication to your cloud provider
+- Team policies, including Policy as Code and Role Based Access Control (RBAC)
 
-{{< choosable cloud gcp >}}
-<p></p>
+The Pulumi Service backend requires no additional configuration after [installing the CLI]({{< relref "/docs/get-started/install" >}}). Pulumi offers this backend hosted online as a free service for individuals with [advanced tiers]({{< relref "/pricing" >}}) for teams and enterprises (with <a href="https://app.pulumi.com/site/trial" target="_blank">free trials</a>. It has successfully undergone multiple security audits including SOC2, pen-testing, and more.
 
-For **Google Cloud Storage**, see [GCP Setup]({{< relref "/docs/intro/cloud-providers/gcp/setup" >}})
-and [Google's Cloud Storage Quickstarts](https://cloud.google.com/storage/docs/quickstarts). See [Application Default Credentials](https://cloud.google.com/docs/authentication/production) for configuring credentials for this backend.
+> To learn more about the Pulumi Service backend's design, including why it doesn't need your cloud credentials, see [Pulumi Service Architecture](#pulumi-service-architecture). If you are interested in the hosting your own instance, see the [Self-Hosting User Guide]({{< relref "/docs/guides/self-hosted" >}}).
 
-{{< /choosable >}}
+Pulumi also lets you manage state yourself using a self-managed backend. This stores state as simple JSON files in AWS S3, Azure Blob Store, Google Cloud Storage, or on your local filesystem. This trades off some amount of reliability for additional control over where metadata is stored. For instance, you will need to manually configure secure access, encryption, and history, and devise your own concurrency control and recovery capabilities. To choose such a backend, use the `pulumi login` command as documented below.
```

Comment:
I think it's important to call out that this is not a paid for service. It doesn't read that way right now